### DISCUSSION TITLE
Multi-Input Monitor / Single Display

### DISCUSSION THREAD

**Discussion by ifoss-me** [hyprwm/Hyprland#1941] [Q&A] [answered]
Hey Hyprland community,

I'm new to you all, but not especially new to Linux. I run Hyprland on Arch using the git AUR package, and have been having some difficulty getting my external monitor to work the way I would like.

In short, I have the monster Dell 32" 8K display, which runs at a 60Hz refresh rate by pairing/merging two DP 1.4 signals. Apologies for not knowing the right graphics/monitor lingo; this is where some my Linux knowledge is weaker. Not entirely sure, therefore, but I think the monitor uses Display Stream Compression to get all this to work.

This happens more or less automatically in GNOME/Wayland when I connect both DP to thunderbolt cables to my laptop (Dell XPS 9320 Plus). In Hyprland, the two DP connections shows as independent displays, at their full resolution but at half the maximum refresh rate (30Hz instead of 60Hz). And intuitively, I get why it would do that by default, but ...

Is there a way to properly merge the two streams into one running at 60Hz? Or, is this even a Hyprland issue versus a lower-level issue that needs massaging?

Thanks for any pointers!

**Comment by vaxerski** [comment, marked as answer]
something like this is a wlroots issue.

**Reply by ifoss-me** [reply]
Thank you! I had a sneaking suspicion this would be the case.
